docs/src/getting-started-vscode-js.md

```diff
@@ -64,27 +64,29 @@ With the VS Code extension you can debug your tests right in VS Code see error m
 
 If your test fails VS Code will show you error messages right in the editor showing what was expected, what was received as well as a complete call log.
 
-<img width="1339" alt="error messaging in vs code" src="https://user-images.githubusercontent.com/13063165/197967016-b4c35689-0c04-4ea3-a288-35b98056efec.png" />
+<img width="1269" alt="error messaging in vs code" src="https://user-images.githubusercontent.com/13063165/212738654-b573b7c9-05be-476f-ab4c-201bf4265bc0.png" />
 
-### Run in Debug Mode
+### Live Debugging
 
-To set a breakpoint click next to the line number where you want the breakpoint to be until a red dot appears. Run the tests in debug mode by right clicking on the line next to the test you want to run. A browser window will open and the test will run and pause at where the breakpoint is set.
+You can debug your test live in VS Code. After running a test with the 'show browser' option checked, click on any of the locators in VS Code and it will be highlighted in the Browser window. You can also edit the locators in VS Code and Playwright will highlight it if it exits and show you if there is more than one result.
```

Comment:
```suggestion
You can debug your test live in VS Code. After running a test with the `Show Browser` option checked, click on any of the locators in VS Code and it will be highlighted in the Browser window. You can also edit the locators in VS Code and Playwright will highlight it if it exits and show you if there is more than one result.
```